### DISCUSSION TITLE
EXTENSION AUTO LOCK

### DISCUSSION THREAD

**Discussion by Shikharishere** [padloc/padloc#731] [Ideas]
I have been use padloc browser extension and android app for quite some time now... And everything works seamlessly... But the browser extension has an auto lock feature with min time of 1 mins to max time of 10 mins... And If you stop auto-lock feature then passwords can be seen by anyone using that device(even if you have closed the browser)... I suggest that if autolock feature is turned off by user(which they can do) then the extension should auto lock after the browser is closed and when the browser is opened again the extensionshould demand password again.....

**Comment by MaKleSoft** [comment]
@Shikharishere The app should definitely be locked when the browser is closed, regardless of the autolock settings. Are you sure you fully closed the browser, and didn't have a window left open somewhere in the background?
